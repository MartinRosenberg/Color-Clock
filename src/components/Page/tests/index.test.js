import { shallow } from 'enzyme'
import React from 'react'

import { Page } from '../index'

test('Page should render correctly', () => {
  const wrapper = shallow(<Page/>)
  expect(wrapper).toMatchSnapshot()
})
