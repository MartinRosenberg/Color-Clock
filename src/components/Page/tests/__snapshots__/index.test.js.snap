// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render correctly 1`] = `
<p>
  Hello world!
</p>
`;
